content/tags.yaml

```diff
@@ -37,4 +37,4 @@
     - Production
     - Suppliers & Production Partners
     - Materials
-    - Equipment
+    - Equipment
```

Comment:
Looks like we lost a new line at the end of the file.....does that break `yaml` files?